tox.ini

```
@@ -13,7 +14,6 @@ commands =
 setenv =
     DJANGO_SETTINGS_MODULE = settings
 deps =
-    django110: Django>=1.10,<1.11
     django111: Django>=1.11,<1.12
```

shouldn't there be explicit entries for 2.0/2.1?
```suggestion
    django111: Django>=1.11,<1.12
    django20: Django>=2.0,<2.1
    django22: Django>=2.1,<2.2
```